DigitalLearningSolutions.Data/Services/PasswordResetService.cs

```diff
@@ -82,7 +83,7 @@ public async Task InvalidateResetPasswordForEmailAsync(string email)
             }
         }
 
-        private string GenerateResetPasswordHash(User user)
+        public string GenerateResetPasswordHash(User user)

```

Comment:
It's a bit organisationally odd to have the reset email generated and dispatched from inside the password-handling service, but the set email generated and dispatched from outside. This is more of a matter of preference though so if you prefer it this way feel free to keep it.